gradle/plugins/common/src/main/kotlin/junitbuild.java-nullability-conventions.gradle.kts

```diff
@@ -27,29 +29,116 @@ nullaway {
 
 tasks.withType<JavaCompile>().configureEach {
 	options.errorprone {
+		disableWarningsInGeneratedCode = true
+		allErrorsAsWarnings = true
+		errorproneArgs.add("-XepOpt:Refaster:NamePattern=^(?!.*Rules\\\$).*") // not complain; reduce after merge.
+		if (System.getenv("IN_PLACE")?.toBooleanStrictOrNull() ?: false) {
+			errorproneArgs.addAll(
+				"-XepPatchLocation:IN_PLACE", // somehow only certain picnic rules apply...
+				"-XepPatchChecks:" +
+					"AddNullMarkedToPackageInfo," +
+					"AmbiguousJsonCreator," +
+					"AssertJNullnessAssertion," +
+					"AutowiredConstructor," +
+					"CanonicalAnnotationSyntax," +
+					"CanonicalClassNameUsage," +
+					"ClassCastLambdaUsage," +
+					"CollectorMutability," +
+					"ConstantNaming," +
+					"DeadException," +
+					"DefaultCharset," +
+					"EagerStringFormatting," +
+					"EmptyMethod," +
+					"EmptyMonoZip," +
+					"ExplicitArgumentEnumeration," +
+					"ExplicitEnumOrdering," +
+					"FluxFlatMapUsage," +
+					"FluxImplicitBlock," +
+					"FormatStringConcatenation," +
+					"IdentityConversion," +
+					"ImmutablesSortedSetComparator," +
+					"IsInstanceLambdaUsage," +
+					"JUnitClassModifiers," +
+					"JUnitMethodDeclaration," +
+					"JUnitNullaryParameterizedTestDeclaration," +
+					"JUnitValueSource," +
+					"LexicographicalAnnotationAttributeListing," +
+					"LexicographicalAnnotationListing," +
+					"MissingOverride," +
+					"MockitoMockClassReference," +
+					"MockitoStubbing," +
+					"MongoDBTextFilterUsage," +
+					"NestedOptionals," +
+					"NestedPublishers," +
+					"NonEmptyMono," +
+					"NonStaticImport," +
+					"OptionalOrElseGet," +
+					"PrimitiveComparison," +
+					"RedundantStringConversion," +
+					"RedundantStringEscape," +
+					"RefasterAnyOfUsage," +
+					"RequestMappingAnnotation," +
+					"RequestParamType," +
+					"Slf4jLogStatement," +
+					"Slf4jLoggerDeclaration," +
+					"SpringMvcAnnotation," +
+					"StaticImport," +
+					"StringJoin," +
+					"TimeZoneUsage"
+			)
+		}
 		val shouldDisableErrorProne = java.toolchain.implementation.orNull == JvmImplementation.J9
 		if (name == "compileJava" && !shouldDisableErrorProne) {
 			disable(
-
-				// This check is opinionated wrt. which method names it considers unsuitable for import which includes
-				// a few of our own methods in `ReflectionUtils` etc.
-				"BadImport",
-
-				// The findings of this check are subjective because a named constant can be more readable in many cases
-				"UnnecessaryLambda",
-
-				// Resolving findings for these checks requires ErrorProne's annotations which we don't want to use
```

Comment:
Assumed this is for the whole block as the block is some kind of pattern here but its a nice trap, assuming the comment only regards to `AnnotateFormatMethod`.